<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SummaryForm</class>
 <widget class="QWidget" name="SummaryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Verdana&quot;;</string>
         </property>
         <property name="text">
          <string>Total Net Returns:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="summary_totalnetreturns">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;color:red;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Verdana&quot;;</string>
         </property>
         <property name="text">
          <string>Most Profitable Equity:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="summary_mostprofitableequity">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Verdana&quot;;</string>
         </property>
         <property name="text">
          <string>Least Profitable Equity:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="summary_leastprofitableequity">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTableWidget" name="summary_returnsTable">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>330</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>30</number>
       </attribute>
       <column>
        <property name="text">
         <string>Equity</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Net Returns (%)</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
